phoenix-core/src/main/java/org/apache/phoenix/coprocessor/IndexRebuildRegionScanner.java

```diff
@@ -109,231 +110,6 @@
 
 public class IndexRebuildRegionScanner extends BaseRegionScanner {
 
-    public static class VerificationResult {
```

Comment:
Why do you need to move this code out from this file? It is a static helper class and only meaningful in the context of index verification and not by itself.

Reply:
Also, for tests, it was 3 level indirection to access the PhaseResult, so I thought of putting it into its own class. :)